<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTransferInvoiceAmount</class>
 <widget class="QDialog" name="DlgTransferInvoiceAmount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfer amount</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Room from</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="EQLineEdit" name="leAmountTo">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="EQLineEdit" name="leRoomCodeFrom">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="EQLineEdit" name="leAmountFrom">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Available amount</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Room to</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Guest</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="EQLineEdit" name="leRoomCodeTo">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Guest</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="EQLineEdit" name="leRoomNameFrom">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="EQLineEdit" name="leGuestFrom">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>38</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPrint">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>39</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="EQLineEdit" name="leRoomNameTo">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="EQLineEdit" name="leGuestTo">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lbBalanceType">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leRoomCodeFrom</tabstop>
  <tabstop>leRoomCodeTo</tabstop>
  <tabstop>leAmountTo</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnPrint</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
